examples/rest-api/database.py

```diff
@@ -0,0 +1,11 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+SQLALCHEMY_DATABASE_URL = "sqlite:///./app.db"
+
+engine = create_engine(SQLALCHEMY_DATABASE_URL, connect_args={"check_same_thread": False})
```

Comment:
```suggestion
engine = create_engine(SQLALCHEMY_DATABASE_URI, connect_args={"check_same_thread": False})
```

Reply:
@sidmohanty11 do you think we should add `connect_args={"check_same_thread": False})` in the Pipeline class as well?

Reply:
Yes, I think so! I've seen some users get this error and once it happened to me during multiple App calls. What do you think?

Ref: https://docs.sqlalchemy.org/en/20/dialects/sqlite.html#threading-pooling-behavior

![Screenshot 2023-11-02 at 6 37 54 PM](https://github.com/embedchain/embedchain/assets/73601258/880ca186-086c-48e2-9ac0-7db63f773e38)